front-end-side/src/Pages/FileUpload.js

```diff
@@ -4,43 +4,54 @@ import axios from "axios";
 const FileUpload = () => {
     const [file, setFile] = useState(null);
     const [fileName, setFileName] = useState(null);
+    const [bank, setBank] = useState(null);
 
     const handleFile = (e) => {
         setFile(e.target.files[0]);
         setFileName(e.target.files[0].name);
+        var e = document.getElementById("banks");
```

Comment:
you could choose another variable name because there is already parameter called e (event) and you use it as new in 12 line

Reply:
You are right, I will fix it